resources/public/api/conf/1.0/sandbox/paye-organisations-matching-request.md

```diff
@@ -0,0 +1,97 @@
+<table>
+    <col width="100%">
+    <thead>
+    <tr>
+        <th>Valid payload</th>
+    </tr>
+    </thead>
+    <tbody>
+    <tr>
+        <td>
+            <p>{&quot;companyRegistrationNumber&quot;:&quot;123456789A&quot;,
+                &quot;employerName&quot;:&quot;Waitrose&quot;,
+                &quot;address&quot;:{
+                    &quot;addressLine1&quot;:&quot;123 Long Road&quot;,
+                    &quot;addressLine2&quot;:&quot;Some City&quot;,
+                    &quot;addressLine3&quot;:&quot;Some County&quot;,
+                    &quot;addressLine4&quot;:&quot;&quot;,
+                    &quot;postcode&quot;:&quot;AB12 3CD&quot}
+                }
+            </p>
+        </td>
+    </tr>
+    </tbody>
+</table>
+
+<table>
+    <col width="25%">
+    <col width="35%">
+    <col width="40%">
+    <thead>
+    <tr>
+        <th>Scenario</th>
+        <th>Payload</th>
+        <th>Response</th>
+    </tr>
+    </thead>
+    <tbody>
+    <tr>
+        <td><p>Successful match</p>
+        <td><p>companyRegistrationNumber = &quot;123456789A&quot;
+            <br/>employerName = &quot;Waitrose&quot;
+            <br/>addressLine1 = &quot;123 Long Road&quot;
+            <br/>addressLine2 = &quot;Some City&quot;
+            <br/>addressLine3 = &quot;Some County&quot;
+            <br/>addressLine4 = &quot;&quot;
+            <br/>postcode = &quot;AB12 3CD&quot;</p></td>
+        <td><p>200 (OK)</p><p>Payload as response example above</p></td>
+    </tr>
+    <tr>
+        <td><p>No match</p></td>
+        <td>
+            <p>Any details that are not an exact match.</p>
+        </td>
+        <td><p>403 (Forbidden)</p>
+        <p>{ &quot;code&quot; : &quot;MATCHING_FAILED&quot;,<br/>&quot;message&quot; : &quot;There is no match for the information provided&quot; }</p></td>
+    </tr>
+    <tr>
+          <td>
+            <p>Missing companyRegistrationNumber &#47; 
+                    employerName &#47; 
+                    addressLine1 &#47; 
+                    addressLine2 &#47; 
```

Comment:
I do not believe that address lines 3 and 4 will be mandatory. Please remove from this list. 

It may also come to light that the postcode is optional however; we can leave as is until we get this confirmed.